modules/aws/cnid-vpc/variables.tf

```diff
@@ -82,3 +82,15 @@ variable "master_sg_ingress_cidr" {
   default     = []
   type        = "list"
 }
+
+variable "master_sg_ssh_ingress_cidr" {
```

Comment:
These variables also need to be inserted into the `cind-vpc` module definition as well right (assuming that was refactored to be in this repo?)

Reply:
Would approved but would like to know the answer to this question first ^

Reply:
Ah I mean https://github.com/conde-nast-international/cnid-infrastructure-core-platform/blob/master/ap-prod/ap-northeast-1/kubernetes/override.tf#L69-L117 because we override there it means we don't need to update it here.

On a side note, would be nice to remove that override.tf file from core-platform and move it here...

Reply:
@tomgco Yes, @katiegamanji already did open this one: https://github.com/conde-nast-international/cnid-infrastructure-core-platform/pull/156/files 
But it's blocked until this one is merged. 

Would be nice, but before that we will need to move this repo to be private.